### DISCUSSION TITLE
Mapping <Tab> to za (open/close fold) effects <C-i> (go back in jumplist). Is this a bug?

### DISCUSSION THREAD

**Discussion by Maltimore** [neovim/neovim#27351] [Q&A] [answered]
I don't want to spam the issues section with something that is not a bug and just me misunderstanding how this mapping works, so I'm posting here first.

To reproduce:
- `nvim --clean`
- perform a jump somewhere, e.g. open a random text file
- `:nnoremap <Tab> za`
- `C-o` to go back in the jump list
- `C-i` to go forward in the jump list

Observe `E490: No fold found`
I don't understand how doing this mapping (which seems completely unrelated) can have that effect.

**Comment by zeertzjq** [comment, marked as answer]
`:h <Tab>`

And, if you are using Konsole (which you mentioned in one of your issues), it doesn't support this.

**Reply by Maltimore** [reply]
Thanks a lot! I learned something new today (the help section you linked an the tmux issue linked there are very informative).
